napari_organoid_counter/_orgacount.py

```diff
@@ -163,7 +277,14 @@ def update_bboxes_scores(self, shapes_name, new_bboxes, new_scores, new_ids):
                     self.pred_ids[shapes_name] = new_pred_ids
 
     def update_next_id(self, shapes_name, c=0):
+        """ Updates the next id to append to result dicts. If input c is given then that will be the next id. """
```

Comment:
There's no check that c doesn't already exist. Is that okay?

Reply:
what kind of check do you mean?

Reply:
`self.next_id[shapes_name] = c` Like what if it's already c (or less) is that okay?